core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+The figure below visually demonstrates high-level Buyer and Seller Order generation and acceptance assuming that a MSA between Buyer and Seller already exists and is recorded on a BPI, and that the commercial state has been synchronized up to this workstep in the commercial business workflow.
+
+<figure>
+  <img
+  src="./images/Baseline-Fig1-Illustrative-Example.png"
+  >
+  <figcaption>Figure 1: Illustrative Example of how the commercial state between Buyer and Seller is synchronized and an Order created.</figcaption>
+</figure>
+
+Without a BPI, both Buyer and Seller must assume that the MSA between them and all its values are correctly represented in the other party’s respective Systems-of-Record. If an order is created based upon the MSA but does not comply with the MSA, it will likely result in extensive manual interactions between Seller and Buyer at one stage or another to re-solve the problem to their mutual satisfaction.  
+
+**[R20]**	The (commercial) counterparties MUST agree on the business process rules which are represented in the business workflows and worksteps in the BPI. 
+
+**[R21]** 	The (commercial) counterparties MUST validate the correctness of a (commercial) state object based on a commercial state change against the transaction business logic in the applicable BPI workflow and workstep.
+
+**[R22]** 	The (commercial) counterparties MUST generate a proof of correctness of a (commercial) state object based on a commercial state change that can be validated against the BPI transaction business logic.
+
+**[R23]** 	Any new (commercial) state between counterparties MUST be recorded on the BPI between them.
+
+**[R24]** 	Any counterparty having received a proof of correctness of a (commercial) state change MUST be able to validate that proof of correctness aginst the BPI between the counterparties.
+
+**[R25]** 	A (commercial) counterparty MUST include a verifiable proof of correctness of the (commercial) state object generated by the commercial state change in the BPI Messages between the transacting counterparties.
+
+### 2.8.2 DLTs and BPI/DLT Abstraction Layers
+
+<figure>
```

Comment:
This figure indicates the Baseline Protocol Core Stack is shared between the two counterparties. Is that true or should it show two instances of the Core Stack, one for each counterparty?

Reply:
@bitwiseguy the operating model is not relevant here ... there could be two instances. It is not relevant or the operating model for that matter since we are discussing the abstraction layers.